internal/model/rwfolder.go

```diff
@@ -1082,3 +1093,10 @@ func removeDevice(devices []protocol.DeviceID, device protocol.DeviceID) []proto
 	}
 	return devices
 }
+
+func moveForConflict(name string) error {
+	ext := filepath.Ext(name)
+	withoutExt := name[:len(name)-len(ext)]
+	newName := fmt.Sprintf("%s (Syncthing conflicted %s)%s", withoutExt, time.Now().Format("2006-01-02 150405"), ext)
```

Comment:
`foo.txt` -> `foo.stconflict-20060102-150405.txt`? I think we need the timestamp in there for multiple conflicts per day.